src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -795,8 +813,8 @@ private sealed class Mapper : MapperBase
             private class OutputCache
             {
                 public long Position;
-                private ValueGetter<ReadOnlyMemory<char>> _imagePathGetter;
-                private ReadOnlyMemory<char> _imagePath;
+                private ValueGetter<VBuffer<byte>> _imageBufGetter;
+                private VBuffer<byte> _imageBuf;
```

Comment:
>_imageBuf [](start = 38, length = 9)

rename to "_image"

Reply:
Changed this in my latest commit. Thanks.